locale/nl.po

```diff
@@ -3425,9 +3413,9 @@ msgid "Failed to load shared library '%s'"
 msgstr "Laden van gedeelde bibliotheek '%s' mislukt"
 
 #: ../src/osx/core/sound.cpp:118
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to load sound from \"%s\" (error %d)."
-msgstr "Laden van bronbestand \"%s\" mislukt."
+msgstr "Laden geluid van \"%s\" mislukt (fout %d)."
 
```

Comment:
You are free to correct things. I wonder however where in a user interface one sees things like windows_left or activeCaption. It looks like variables or keywords to me. 

Ciao,

Gideon

~~o
<#\,
()/()

(Verstuurd vanaf mijn iPad air)

> Op 2 mei 2015 om 11:36 heeft Dimitri Schoolwerth <notifications@github.com> het volgende geschreven:
> 
> In locale/nl.po:
> 
> >  msgid "Failed to load sound from \"%s\" (error %d)."
> > -msgstr "Laden van bronbestand \"%s\" mislukt."
> > +msgstr "Laden geluid van \"%s\" mislukt (fout %d)."
> >  
> Personally I'd prefer "Laden van geluid \"%s\" [...]" or "Laden geluid vanaf \"%s\" [...]" . It sounds a little less 'krom' IMHO.
> 
> —
> Reply to this email directly or view it on GitHub.
>